site/content/en/docs/tasks/run/jobs.md

```diff
@@ -18,7 +18,7 @@ Make sure the following conditions are met:
 - A Kubernetes cluster is running.
 - The kubectl command-line tool has communication with your cluster.
 - [Kueue is installed](/docs/installation).
-- The cluster has [quotas configured](/docs/tasks/administer_cluster_quotas).
+- The cluster has [quotas configured](/docs/tasks/manage/administer_cluster_quotas).
```

Comment:
The old wrong link `/docs/tasks/administer_cluster_quotas` is also present in other pages. Please fix them all in this PR

Reply:
Right, did that in 25009bb1.

It seems that the actual root cause was due to invalid target URLs in the Netlify server-side redirects, which redirected to `/doc/...` instead of `/docs/...`. I updated those in 916281b4.

In total, only the last change might be necessary, however, updating the links on the individual pages saves the browser one redirect (unless you prefer the older, shorter, URLs).